lesson_01/main.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("Hello, world, :face:")
+}
```

Comment:
If you can't see emoji in terminal, you can try to write your data to file
just for example:

file, _ := os.Create("response.txt")
file.WriteString(**result**)

where **result** is your string with emoji inside